cypress/tests/ImageElement.spec.ts

```diff
@@ -27,6 +29,19 @@ describe("ImageElement", () => {
 
   describe("Fields", () => {
     describe("Rich text field", () => {
+      it("should update the document selection when the user focuses on the field", () => {
+        addImageElement();
+        getDocSelection().then(([from]) => {
+          // The selection should be just after the inserted element
+          expect(from).to.be.greaterThan(5);
+        });
+        focusElementField("caption");
```

Comment:
Could we make this test a little more thorough by also selecting another empty text field?